### DISCUSSION TITLE
"Error fetching version information" under system info

### DISCUSSION THREAD

**Discussion by ahaspharos** [tbnobody/OpenDTU#493] [General]
Hello.

Since the last firmware update i have an issue on the system info tab
![image](https://user-images.githubusercontent.com/8578356/210609043-acad6bbf-02dc-4685-8457-1a07148d5181.png)
before, the was a notification for a new firmware update. now, there is an error message.

also the link to the commit doesn't work
![image](https://user-images.githubusercontent.com/8578356/210609253-8c2f2195-f1ab-44fc-ab8d-06df76a5ad4c.png)

anybody an idea? 
thanks

**Comment by tbnobody** [comment]
This screenshot shows a githash: 7df8ed3 which is not a hash which belongs to this repo. it's most likely that you changed some code and added commits. In this case the update check which is related to the current compiled git hash will not work

**Reply by ahaspharos** [reply]
ah ok. thank you. i have uncommited local changes (com port ...) that's the issue

**Reply by tbnobody** [reply]
You should not commit changes in the serial port. Just use the platformio_override.ini to specify your serial port. this file is included in the .gitignore file and will not be added to the git index.
